classes/set_subclass.py

```diff
@@ -0,0 +1,31 @@
+class PositiveSet(set):
+    """PositiveSet class documentation"""
+
+    def __init__(self, input_set):
+        """constructor"""
+        super(PositiveSet, self).__init__({number for number in set(input_set)

```

Comment:
Так было в питоне 2. В третьем можно проще
```suggestion
        super().__init__({number for number in set(input_set)

```